src/app/landing-page/landing-page.component.spec.ts

```diff
@@ -0,0 +1,63 @@
+import { Component } from '@angular/core';
+import { TestBed } from '@angular/core/testing';
+
+import { Broadcaster } from 'ngx-base';
+import { AuthenticationService } from 'ngx-login-client';
+
+import { LoginService } from '../shared/login.service';
+import { LandingPageComponent } from './landing-page.component';
+
+import { createMock } from 'testing/mock';
+
+import {
+  initContext,
+  TestContext
+} from 'testing/test-context';
+
+
+@Component({
+  template: '<alm-landing-page></alm-landing-page>'
+})
+class HostComponent { }
+
+describe('LandingPageComponent', () => {
+  type Context = TestContext<LandingPageComponent, HostComponent>;
+
+  let mockAuthenticationService: AuthenticationService;
```

Comment:
This looks unused, is it?

Reply:
I used it when I declare the AuthenticationService provider a bit later on, I believe this is necessary ? Line 40.

Reply:
You redeclare a variable by the same name on line 41, shadowing this first declaration, and this new variable is returned at the end of its scope (the factory anonymous function). Later when you access this instance on line 51 you do so via `TestBed.get`. So there are no apparent accessors of this declaration on line 26.